test/machinarium/test_sleeplock.c

```diff
@@ -0,0 +1,45 @@
+
+#include <machinarium.h>
+#include <sleep_lock.h>
+#include <odyssey_test.h>
+
+mm_sleeplock_t global_lock;
+
+static void
+test_coroutine(void *arg)
+{
+	uint64_t* value = (uint64_t*)arg;
+	for (int i = 0; i < (1 << 22); i++){
```

Comment:
I wonder, how much time this test takes to execute?

Reply:
.2 sec on my machine

Reply:
Thats fine for me, just curious what if run this in CPU limited env